components/Icons/icons/icon_arrow.tsx

```diff
@@ -0,0 +1,10 @@
+export const ArrowIcon = (props: any) => (
```

Comment:
It would be good to change file name to `IconArrow.tsx` according to the convention.